packages/rancher-logging/package.yaml

```diff
@@ -1,6 +1,6 @@
 url: https://kubernetes-charts.banzaicloud.com/charts/logging-operator-3.9.4.tgz
 packageVersion: 00
-releaseCandidateVersion: 03
+releaseCandidateVersion: 04
```

Comment:
This should actually be
```
packageVersion: 01
releaseCandidateVersion: 00
```
since this is a v2.6/v2.5.9 chart

Reply:
> @dbason would you mind rebasing one more time?

@nickgerace done, and the version updated :)